### ISSUE TITLE
[DRAFT] VAMC QA/508 Check

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#9553]
# Team Name
Facilities

# Product/Feature Name
VAMC template

## Checkpoint Stage
- [x] Other: Requesting 508 and QA review for a variety of VAMC front end 508 defect resolutions and small UI enhancements which are now available in **Production**

## Point of contact
Michelle Middaugh

## Production URL 
https://www.va.gov/find-locations
No credentials needed 

## User stories for testing

[#4210](https://github.com/department-of-veterans-affairs/va.gov-team/issues/4210) 
- [ ] When a Veteran visits https://www.va.gov/pittsburgh-health-care/stories and uses a screen reader device to navigate the page, story titles should be presented in h2 to support navigation and structural understanding. 
- [ ] Visually, the titles should be displayed with h3 styling. 
- [ ] If the code is inspected, it should not skip heading levels and should use font-utility for styling, consistent with the [typography design system](https://design.va.gov/design/typography#headings-and-accessibility) 

[#8065](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8065)
- [ ] When a Veteran visiting [Stories content on Pittsburgh health care](https://www.va.gov/pittsburgh-health-care/stories) sees the most recent story first, beginning after the spotlight story.  
- [ ] Top story and spotlight story should not be the same, in the event that the spotlight story is also the most recent. 

[#8155](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8155)
- [ ] When a Veteran visits https://www.va.gov/pittsburgh-health-care/, stories and events can be viewed near the bottom of the home page (between blue "In the spotlight at VA Pittsburgh health care" and gray "Get updates from VA Pittsburgh health care" boxes.)

[#4255](https://github.com/department-of-veterans-affairs/va.gov-team/issues/4255) 
- [ ] Given a Veteran visits https://www.va.gov/pittsburgh-health-care/ in browser using a screen reader device and navigates to the h1 above the duotone photo of the medical center: when the device reads the content of the page starting at the h1, the alt-text for the photo should help the reader understand it is a decorative element which is not required for understanding the content. 

[#3448](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3448) 
- [ ] Given a sighted user visits https://staging.va.gov/pittsburgh-health-care/ in browser and selects Locations to open the accordion, the Veteran can clearly understand how the content is related because labels are not repeated or able to be selected once active. 

[#5993](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5993)
- [ ] Given a Veteran visits https://www.va.gov/pittsburgh-health-care/events/veterans-town-hall-0/ and uses the keyboard to navigate to "Add to calendar", link will receive keyboard focus. 

[#5961](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5961)
Given a Veteran uses a screen reader to visit one of the following pages, there should not be any confusion created by aria-labelledby attributes which reference a missing ID. 
- [ ] https://www.va.gov/pittsburgh-health-care/university-drive-campus-map/
- [ ] https://www.va.gov/pittsburgh-health-care/internships-fellowships/forms/
- [ ] https://www.va.gov/pittsburgh-health-care/research/safety-security/animal-research/

.

## Next steps
- [ ] If the linked issue within the test case above is resolved, the QA or Accessibility specialist will close the related issue. 
- [ ] If the linked issue within the test case above remains unresolved, the QA or Accessibility specialist will comment in the related issue. 
- [ ] When all user stories above have been retested, the QA or Accessibility specialist will comment in this issue to confirm completion, tagging @mmiddaugh so that the team can coordinate any needed followup. 
- [ ] If additional work is needed, the Facilities team will coordinate another review after work is updated 
- [ ] Once all issues are addressed, the ticket is closed with a final comment, “All issues are addressed”, @-comment developer who opened issue, and close ticket/issue.

**Comment by mmiddaugh** [comment]
I have tagged the appropriate people in individual issues and so will close this one.
